packages/datagrid/src/components/default-header-renderer/quality-bar.component.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
```

Comment:
The global file layout doesn't respect our guideline.
src/components/QualityBar/QualityBar.component.js